docs/cli/cli-guide.md

```diff
@@ -0,0 +1,179 @@
+---
+title: CLI Guide
+---
+
+## Authentication
+
+The `auth` command allows the CLI to authenticate with Planet servers. Before
+any other command is run, the CLI authentication should be initiated with
+
+```console
+$ planet auth init
+```
+
+To store the authentication information in an environment variable, e.g.
+for passing into a Docker instance:
+
+```console
+$ export PL_API_KEY=$(planet auth value)
+```
```

Comment:
this seems like a good time to mention that the presence of the environment variable overrides changes to the stored auth with planet auth init